### PR TITLE
docs: Port guides to new format

### PR DESCRIPTION
**Pull request by mqasimsarfraz**
Port guides to new format. Testing instructions available at: https://github.com/inspektor-gadget/inspektor-gadget/pull/3254#issue-2436454494

### TODO:
- [ ] Port other guides

### CODE REVIEW

File: gadgets/trace_tcpdrop/README.mdx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        test-trace-tcpdrop                                                          172.17.0.2:37838                                     1.1.1.1:443                                          4026533225                               wget                                               854278                                  854278 tcp_established                         ACK                                     SKB_DROP_REASON_QDISC_DROP              2024-08-02T12:35:20.597218523+02:00
+        test-trace-tcpdrop                                                          172.17.0.2:37838                                     1.1.1.1:443                                          4026533225                               wget                                               854278                                  854278 tcp_fin_wait1                           FIN|ACK                                 SKB_DROP_REASON_QDISC_DROP              2024-08-02T12:35:25.659670258+02:00
+        ```
+    </TabItem>
+</Tabs>
+
+### List of drop reasons
```
Comment (mauriciovasquezbernal) [inline gadgets/trace_tcpdrop/README.mdx:102]: I think it shouldn't be part of "guide" but more general documentation of the gadget.

**Review by mauriciovasquezbernal — APPROVED** [review]
I did a quick check. Some small issues but LGTM already.

File: gadgets/trace_bind/artifacthub-pkg.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        - linux/arm64
+keywords:
+    - gadget
+links:
+    - name: source
+      url: "https://github.com/inspektor-gadget/inspektor-gadget/"
+install: |
+    # Run
+    ```bash
+    sudo IG_EXPERIMENTAL=true ig run ghcr.io/inspektor-gadget/gadget/trace_bind:latest
```
[the remark this answers is not in the document]
Comment (mqasimsarfraz) [reply gadgets/trace_bind/artifacthub-pkg.yml:26]: Good Catch!

File: gadgets/trace_bind/README.mdx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+### `--ignore_errors`
+Show only events where the bind succeeded. By default all events are traced.
+
+### `--pid`
+Show only bind events generated by this particular PID. By default all events are traced.
+
+## Guide
+
+This example shows how to use this gadget.
```
Comment (burak-ok) [inline gadgets/trace_bind/README.mdx:39]: This text is almost always repeated directly as the first sentence under 'guide'.

Is this necessary?

File: docs/gadgets/trace_exec.mdx [hunk 4]
```diff
@@ -0,0 +1 @@
+../../gadgets/trace_exec/README.mdx
```
[the remark this answers is not in the document]
Comment (mauriciovasquezbernal) [reply docs/gadgets/trace_exec.mdx:1]: Exactly, we want to keep the gadget documentation as close as possible to the source, so using a symlink is a solution for that.

Reasons to keep the readme closer:
- Easy to discover and update when there are changes to the gadget
- We'll move gadgets to a different repository later on 
- We could package the readme within the image in the future

File: gadgets/trace_capabilities/artifacthub-pkg.yml [hunk 5]
```diff
@@ -19,11 +19,11 @@ keywords:
     - gadget
 links:
     - name: source
-      url: "https://github.com/inspektor-gadget/inspektor-gadget/"
+      url: "https://github.com/inspektor-gadget/inspektor-gadget/tree/main/gadgets/trace_capabilities"
 install: |
     # Run
     ```bash
-    sudo IG_EXPERIMENTAL=true ig run ghcr.io/inspektor-gadget/gadget/trace_capabilities:latest
+    sudo -E ig run ghcr.io/inspektor-gadget/gadget/trace_capabilities:latest
```
[the remark this answers is not in the document]
Comment (burak-ok) [reply gadgets/trace_capabilities/artifacthub-pkg.yml:26]: I don't think so. No env variables are needed

**Comment by mqasimsarfraz** [comment]
Thanks for the reviews! The failing CI isn't related to this change!

**Pull request merged**